README.md

```diff
@@ -79,3 +79,57 @@ Running multiple commands works the same way as the standard `run` command:
 Multiple devices can also be specified this way.
 
 The [release](.github/workflows/release.yml) workflow contains an example usage of this action.
+
+### Python packages
+
+This action offers sideloading your Python packages that you want to use in emulated Linux. You can select any package from the pypi repository
+or from the git repository.
+
+For example:
+
+```yaml
+- uses: antmicro/renode-linux-runner-action@v0
+  with:
+    shared-dir: ./shared-dir
+    renode-run: python --version
+    python-packages: |
+      git+https://github.com/antmicro/pyrav4l2.git
+      pytest
+```
+
+You can also pass the specific version requirement:
+
+```yaml
+- uses: antmicro/renode-linux-runner-action@v0
+  with:
+    shared-dir: ./shared-dir
+    renode-run: python --version
+    python-packages: |
+      git+https://github.com/antmicro/pyrav4l2.git@3c071a7
+      pytest==5.3.0
+      pyyaml>=5.3.1
+```
+
+The action will download all selected packages and their dependencies and install them later in the emulated Linux environment.
+
+## Git repositories
+
+If you want to use some files that are outside of your repository, you can sideload them to the emulated Linux with the `repos` argument:
+
+```yaml
+- uses: antmicro/renode-linux-runner-action@v0
+  with:
+    shared-dir: ./shared-dir
+    renode-run: python --version
+    repos: https://github.com/antmicro/pyrav4l2.git
+```
+
+You can also specify the catalog name into which you want to clone the repository:
```

Comment:
```suggestion
You can also specify the directory into which you want to clone the repository:
```
Can it be a path with slashes? If so, maybe use _path_ instead of _directory_.